### ISSUE TITLE
Is this plugin still working properly?

### ISSUE THREAD

**Issue by SWFLInc** [Prospress/woocommerce-subscriptions-preserve-billing-schedule#10]
Hello there,

I just installed and created a new manual subscription to test and the next payment date was still getting changed based on the day it was payed. I reached out to WooCommerce support and they tested it on their end and said it didn't seem to be working properly either and suggested I ask here.

Thank you for your contribution to this project.

**Comment by james-allan** [comment]
Hi @SWFLInc. 

Yes, this plugin does still work as intended. I just ran a test and paying for a renewal 10 minutes after it was created calculates a new next payment date from the previous payment schedule, not a new date 10 minutes later.

If you're still having this problem can you explain how you're testing this and give some examples of the subscription's start date, billing schedule (monthly, weekly), original next payment dates, paid dates and then the incorrect new next payment?

**Comment by SWFLInc** [comment]
Hi @james-allan 

I created a yearly manual subscription with a start date of August 1, 2020, set the next payment date to August 1, 2021, added my product to the subscription, selected create a pending parent order and created the subscription.

Manual subscription before changing parent order status to complete: https://snipboard.io/6o7E4Z.jpg and https://snipboard.io/cUzIG1.jpg.

Then I went in and set the order status to completed, as we have manually received payment and it changed the subscription schedule to next payment date of August 6, 2021. See https://snipboard.io/xU185Q.jpg and https://snipboard.io/GBK2Ac.jpg.

I must be doing this process wrong then I assume?
